kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -110,15 +110,47 @@ class ResidualBasedNewtonRaphsonStrategy
     {
     }
 
+    /**
+     * @brief Default constructor. (with parameters)
+     * @param rModelPart The model part of the problem
+     * @param ThisParameters The configuration parameters
+     */
+    explicit ResidualBasedNewtonRaphsonStrategy(ModelPart& rModelPart)
+        : ResidualBasedNewtonRaphsonStrategy(rModelPart, ResidualBasedNewtonRaphsonStrategy::GetDefaultParameters())
```

Comment:
I don't think this works, I had it first like this before I moved to static functions

Reply:
I use static functions

Reply:
Ups, I don't